mlir/include/Driver/CompilerDriver.h

```diff
@@ -82,7 +85,7 @@ struct CompilerOptions {
 
 struct CompilerOutput {
     typedef std::unordered_map<Pipeline::Name, std::string> PipelineOutputs;
-    std::string objectFilename;
+    std::string outputFilename = "-";
```

Comment:
Are we always outputting to stdout? Or what do you mean by this?

Reply:
no I should have removed this. Sorry for that. Fixed it.